test/tests/userobjects/particle_initializer/uniform_grid/uniform_grid.i

```diff
@@ -30,7 +27,7 @@
     charge = 1
```

Comment:
I can't provide a suggestion to replace `[initializer]` with `[particle_initializer]` (too far from your changes), but please make sure you do so. I think it helps distinguish with `[velocity_initializer]`